File: src/Python/Parcial2/Geometria/triangulo.py
from poligono import Poligono

class Triangulo(Poligono):

    def __init__(self, numLados, base, altura):
        super().__init__(numLados)
        self._base = base
        self._altura = altura

    def calcularArea(self):
        return (self._base * self._altura) / 2

    def calcularPerimetro(self):
        return self._base * 3  # suponiendo triángulo equilátero

File: src/C#/Parcial2/Geometria/Triangulo.cs
using System;

public class Triangulo : Poligono
{
    protected double @base;
    protected double altura;

    public Triangulo(int numLados, double b, double altura)
    {
        this.numLados = numLados;
        this.@base = b;
        this.altura = altura;
    }

    public override double calcularArea()
    {
        return (@base * altura) / 2;
    }

    public override double calcularPerimetro()
    {
        return @base * 3; // suponiendo triángulo equilátero
    }
}